spec/models/expense_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe Expense, type: :model do
+  before(:each) do
```

Comment:
- [ ] Please make sure that The App has integration tests for all the most important components of the RoR application. 💯 👍🏻